l10n/en/firefox/products.ftl

```diff
@@ -4,63 +4,73 @@
 
-firefox-products-firefox-monitor = { -brand-name-firefox-monitor }
+## Monitor
 
+firefox-products-mozilla-monitor = { -brand-name-mozilla-monitor }
 firefox-products-see-if-your-personal-information = See if your personal information has been compromised in a corporate data breach, and sign up for future alerts.
 firefox-products-check-for-breaches = Check for breaches
-firefox-products-sign-up-for-breach-alerts = Sign up for breach alerts
 
-# Obsolete string
-firefox-products-firefox-browsers = { -brand-name-firefox } browsers
+## Relay
+
+firefox-products-relay = { -brand-name-firefox-relay}
+firefox-products-protect-your-real = Protect your real email address to help control your inbox.
+firefox-products-get-relay = Get { -brand-name-firefox-relay}
+
+## VPN
 
-firefox-products-firefox = { -brand-name-firefox }
-firefox-products-get-the-browsers-that-block = Get the browsers that block 2000+ data trackers automatically. Enhanced Tracking Protection comes standard in every { -brand-name-firefox } browser.
-firefox-products-desktop = Desktop
-firefox-products-android = { -brand-name-android }
-firefox-products-ios = { -brand-name-ios }
-firefox-products-see-all-browsers = See all browsers
 firefox-products-mozilla-vpn = { -brand-name-mozilla-vpn }
 firefox-products-surf-stream-and-get-work-done = Surf, stream and get work done on servers in over 30 countries for a secure internet connection with a new perspective.
 firefox-products-get-mozilla-vpn = Get { -brand-name-mozilla-vpn }
-firefox-products-relay = { -brand-name-firefox-relay}
-firefox-products-protect-your-real = Protect your real email address to help control your inbox.
-firefox-products-learn-more-about-relay = Learn more about { -brand-name-firefox-relay }
 
-firefox-products-send-your-large-files-and = Send your large files and sensitive documents safely, up to 2.5G.
-firefox-products-send-a-file = Send a file
+## MDN Plus
+
+firefox-products-mdn-plus = { -brand-name-mdn-plus }
+firefox-products-resources-for-developers = Resources for developers, by developers. Support MDN and make it your own with collections, notifications, and playgrounds.
+firefox-products-support-mdn = Support { -brand-name-mdn }
+
+## Thunderbird
+
+firefox-products-thunderbird = { -brand-name-thunderbird }
+firefox-products-access-all = Access all your email, calendars, and contacts in one fast app. Filter and organize them the way you like.
+firefox-products-download-thunderbird = Download { -brand-name-thunderbird }
+
+## Fakespot
+
+firefox-products-fakespot = { -brand-name-fakespot }
+# scam artists are people who are trying to sell fakes or steal money
+# 'weeding' is a gardening term, here it is used to mean "removing unwanted things without touching the good ones"
+firefox-products-fakespot-has-your = Fakespot has your back when you’re shopping online - weeding out scam artists and finding you the best deals.
```

Comment:
```suggestion
firefox-products-fakespot-has-your = Fakespot has your back when you’re shopping online — weeding out scam artists and finding you the best deals.
```

The monospaced font doesn't show it well but this is an em-dash "—" to be consistent with the dash right next to it in Pocket content.